exports/Disconnect-Interaction.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
Given the way the blueprint has been written, I'd remove the "exports" directory from the blueprint and move this file ("Disconnect-Interaction.json") into the parent directory.  This will make it easier for customers implementing the blueprint to find the data action export.  This will also make it consistent with what you've done with the Blacklist workflow export.